124-exec-engine.Rmd

```diff
@@ -0,0 +1,190 @@
+---
+title: About `exec` engine
+editor: visual 
```

Comment:
> BTW, I'm using the visual markdown editor to edit the Rmd file, to make it easier for readers to view the content (avoid horizontal scrolling on Github).

I added this so that when opening in the IDE it uses the visual editor by default

Reply:
Since this repo is maintained mostly by you and me, it probably doesn't make much sense to config the IDE on a per-file basis. I have wanted to hard-wrap all Rmd files for years but the visual editor didn't exist back then. From now on, I'd like to use the visual editor for all Rmd files in this repo and hard-wrap lines.

Reply:
I did not find the project setting to run on the visual editor by default when opening a Rmd file. This setting on per file basis does that. If I open the file, it will switch to visual mode directly. 

I added that initially by convenience only. but do you know the global setting ? 

If we do not add that, we just need to manually switch to visual editor. I'll add a note in README that RStudio visual editor is used for all the file in the repo.